Composer/packages/electron-server/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "@bfc/electron-server",
+  "license": "MIT",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0",
+  "description": "Electron wrapper around Composer that launches Composer as a desktop application.",
+  "main": "./build/main.js",
+  "engines": {
+    "node": ">=12"
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "clean": "rimraf build && rimraf dist",
+    "copy-plugins": "node scripts/copy-plugins.js",
+    "copy-templates": "node scripts/copy-templates.js",
+    "dist": "node scripts/electronBuilderDist.js",
+    "dist:full": "yarn clean && yarn build && yarn copy-templates && yarn run pack && yarn copy-plugins && yarn dist",
+    "pack": "node scripts/electronBuilderPack.js",
+    "start": "cross-env NODE_ENV=development electron .",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint --quiet --ext .js,.jsx,.ts,.tsx ./src",
+    "lint:fix": "yarn lint --fix",
+    "start:electron": "./node_modules/.bin/electron --inspect=7777 --remote-debugging-port=7778 .",
+    "start:electron:dev": "cross-env ELECTRON_TARGET_URL=http://localhost:3000/ npm run start:electron"
+  },
+  "devDependencies": {
+    "@types/archiver": "^3.0.0",
+    "@types/body-parser": "^1.17.0",
+    "@types/compression": "^1.0.1",
+    "@types/cookie-parser": "^1.4.1",
+    "@types/debug": "^4.1.2",
+    "@types/express": "^4.16.1",
+    "@types/form-data": "^2.2.1",
+    "@types/globby": "^9.1.0",
+    "@types/http-errors": "^1.6.1",
+    "@types/jest": "^24.0.9",
+    "@types/jsonwebtoken": "^8.3.3",
+    "@types/lodash": "^4.14.146",
+    "@types/lru-cache": "^5.1.0",
+    "@types/mock-fs": "^3.6.30",
+    "@types/morgan": "^1.7.35",
+    "@types/node": "^13.7.7",
+    "@types/rimraf": "^2.0.2",
+    "electron": "8.0.2",
+    "electron-builder": "^22.4.1",
+    "fs-extra": "^9.0.0",
+    "js-yaml": "^3.13.1",
+    "mock-fs": "^4.10.1",
+    "nodemon": "^1.18.11",
+    "path": "^0.12.7",
+    "rimraf": "^2.6.3",
+    "source-map-explorer": "^2.1.0",
+    "ts-node": "^8.4.1"
+  },
+  "dependencies": {
+    "@bfc/server": "*",
+    "debug": "4.1.1",
+    "fix-path": "^3.0.0",
+    "lodash": "^4.17.15"
+  },
+  "build": {
```

Comment:
is it possible for this to live outside of the package.json?

Reply:
Yes, I can extract this out into a json 👍